src/aggregation/agg_req_with_accessor.rs

```diff
@@ -110,7 +131,11 @@ impl MetricAggregationWithAccessor {
             | MetricAggregation::Min(MinAggregation { field: field_name })
             | MetricAggregation::Stats(StatsAggregation { field: field_name })
             | MetricAggregation::Sum(SumAggregation { field: field_name }) => {
-                let (accessor, field_type) = get_ff_reader_and_validate(reader, field_name)?;
+                let (accessor, field_type) = get_ff_reader_and_validate(
+                    reader,
+                    field_name,
+                    Some(get_numeric_or_date_column_types()),
```

Comment:
does it make sense to allow sum for dates?

Reply:
since the baseline with 1970 is arbitrary, no doesn't make sense